release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -707,6 +707,29 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 
 * Previously, egress IPs on vSphere were picked up by the vSphere cloud provider within the kubelet. These were unexpected by the certificate signing requests (CSR) approver. Consequently, nodes with egress IPs would not have their CSR renewals approved. This update allows the CSR approver to account for egress IPs. As a result, nodes with egress IPs on vSphere SDN clusters now continue to function and have valid CSR renewals. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1860774[*BZ#1860774*])
 
+* Previously, worker nodes failed to launch, and the installer failed to generate URL images due to the broken path defaulting for the disk image and incompatible changes in the Google Cloud Platform (GCP) SDK. As a result, the machine controller was unable to create machines. This fix repairs the URL images by changing the base path in the GCP SDK. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2009111[*BZ#2009111*])
+
+* Previously, the machine would freeze during the deletion process due to a lag in the vCenter's `powerOff` task. VMware showed the machine to be powered off, but {product-title} reported it to be powered on, which resulted in the machine freezing during the deletion process. This update improves the `powerOff` task handling on the vSphere to be checked before the destroy task is created, which prevents the machine from freezing during the deletion process. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011668[*BZ#2011668])
+
+* Previously, after installing or updating {product-title}, the value of the metrics showed one pending CSR after the last CSR was reconciled. This resulted in the metrics reporting one pending CSR when there should be no pending CSRs. This fix ensures the pending CSR count is always valid post-approval by updating the metrics at the end of each reconcile loop. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2013528[*BZ#2013528])
+
+* Previously, AWS checked for credentials when the `cloud-provider` flag was set to empty string. The credentials were checked by making calls to the metadata service, even on non-AWS platforms. This caused latency in the ECR provider startup and AWS credential errors logged in all platforms, including non-AWS. This fix prevents the credentials check from making any requests to the metadata service to ensure that credential errors are no longer being logged. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2015515[*BZ#2015515])
+
+* Previously, the Cluster Autoscaler Unschedulable pods showed a severity warning alert that suggested it required developer intervention. This alert is informational and does not describe a problematic condition that requires developer intervention. This issue resulted in the Cluster Autosaver Unschedulable pods not meeting the warning alert criteria. This fix changes the Cluster Autosaver Unschedulable pods by lowering its severity info. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2025230[*BZ#2025230])
```

Comment:
"lowering its severity info" doesn't sound right. Should it be just "lowering its severity.", or perhaps "lowering its severity level to `info`"?